libs/api/domains/vehicles/src/lib/api-domains-vehicles.service.ts

```diff
@@ -163,6 +163,48 @@ export class VehiclesService {
     }
   }
 
+  private async isAllowedMileageRegistration(
+    auth: User,
+    permno: string,
+  ): Promise<boolean> {
+    const res = await this.getVehicleDetail(auth, {
+      clientPersidno: auth.nationalId,
+      permno,
+    })
+
+    // String of owners where owner can delegate registration.
+    const allowedCoOwners = process.env.VEHICLES_ALLOW_CO_OWNERS?.split(', ')
```

Comment:
Safer to split on `,` and trim.

```suggestion
    const allowedCoOwners = process.env.VEHICLES_ALLOW_CO_OWNERS?.split(',').map(i => i.trim())
```